.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+exclude=alembic/versions,.venv
```

Comment:
We can delete `.venv` once we get rid of `requirements.txt` right?

Reply:
Addressed in: [5d786ae](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/5d786ae38cdc522cec9669c3bea0aacb2149d523)

Reply:
This `.venv` is generated in the github actions itself from the poetry install action, `snok/install-poetry@v1`. It is automatically generated from the action and I use it for caching so the next time the github actions run after installing the dependencies its much faster. Hence I ignore it in flake8 here

Reply:
RE-addressed in: [b103e18](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/b103e185667d42c426411f4f79f8c426d16f4d33)

Add .vev back.